### DISCUSSION TITLE
桥本有菜个人信息刮削不到

### DISCUSSION THREAD

**Discussion by chn-zhong** [sqzw-x/mdcx#144] [Q&A]
rt
可能是因为改名叫新有菜的原因，桥本有菜刮削完，在jellyfin中可以显示头像，但是个人信息打不开，通过数据库添加同样的问题，求大佬解答

**Comment by sqzw-x** [comment]
修改姓名映射表或者更改刮削源.
另外建议使用日文名, wiki 和数据库信息更全, 中文名的话就只能手动调整映射表
